.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
Could we delete this file please - using yarn on this project and prefer to have lock files in place

Reply:
npm will ignore lock files in dependencies:
```
One key detail about package-lock.json is that it cannot be published, and it will be ignored if found in any place other than the toplevel package.
```
from: https://docs.npmjs.com/files/package-lock.json

Reply:
This simply disables the lockfile entirely - meaning npm will not generate one. That is not behaviour we want nor do we want to use npm either - this project uses yarn.